restli-server/src/test/java/com/linkedin/restli/internal/server/model/SampleResources.java

```diff
@@ -759,4 +761,16 @@ public void AttachmentParamsIncorrectDataTypeAction(@RestLiAttachmentsParam Stri
 
   @RestLiCollection(name = "collectionComplexKeyTask")
   class CollectionComplexKeyTask extends ComplexKeyResourceTaskTemplate<EmptyRecord, EmptyRecord, EmptyRecord> {}
+
+  @RestLiCollection(name="lucky",keyName="dayOfWeek")
+  public class fooBarFinderResource extends UnstructuredDataCollectionResourceTemplate<Integer>
```

Comment:
Given that this file is bloated with resource classes, I'd suggest naming this something meaningful to clarify what it's testing. I'd suggest `_____Resource` where the prefix describes what scenario it covers (e.g. `NonRecordFinderResource` or `UnsupportedReturnTypeFinderResource`)